common/lib/capa/capa/tests/test_files/extended_hints_checkbox.xml

```diff
@@ -0,0 +1,108 @@
+<problem schema="edXML/1.0">
```

Comment:
Oh yeah, that's some sort of leftover that does nothing. I'll take them all out.